### DISCUSSION TITLE
How do you sync and deal with multiple devices?

### DISCUSSION THREAD

**Discussion by topaxi** [nvim-neorg/neorg#843] [Q&A] [answered]
I recently started using Neorg and I love it! One thing I currently struggle with a bit is, that there is no official way to sync notes between devices (automatic git commits/pushes/merges would be amazing :star_struck:).

Maybe I'm a minority here and you don't have this. I'm dealing with the following three devices I'm using on a regular basis:

- Notebook for work
- Notebook for personal projects
- Workstation at home for either work or personal projects

I'm currently syncing the notes each time I switch devices manually via git. But this gets a bit tiring with time.

How do you deal with similar situations to this? :thinking:

**Comment by BSathvik** [comment]
I just use Google Drive sync. I use it for backup, but I imagine syncing to multiple devices would work pretty well. No extra work required other than setting up your neorg workspace to sync. 

You also get protection against accidental file deletion. Drive keeps your files around for 30 days in Trash before permanently deleting them. 

The only downside to this approach of course is that you don't have a revision history. I've been thinking about setting up automatic git commit-push on BufWrite but haven't gotten around to it.

**Comment by inhake2** [comment, marked as answer]
I use [syncthing](https://syncthing.net/) without relay server and through a wireguard vpn (android phone included). It supports file versioning in many ways and has a very nice GUI.

**Comment by topaxi** [comment]
Thanks for the input :) I'm actually moving from (manual) git to `syncthing`. Seems like a neat solution and as I'm the only "editor" for my notes, I do not need anything to resolve potential conflicts.

**Comment by turkishfilms** [comment]
I have thsi problem i just  moved my whole GTD second brain from apple notes on my iphone to neorg, but i dont want to whip out my laptop ever time i need to capture something.

**Reply by lukebarton** [reply]
I used to be obsessed with having everything everywhere but now I use my phone to capture things into Apple Notes and then transfer them to my main system later.
